dev/tools/lib/roll_dev.dart

```diff
@@ -40,7 +41,7 @@ void main(List<String> args) {
     kCommit,
     help: 'Specifies which git commit to roll to the dev branch.',
     valueHelp: 'hash',
-    defaultsTo: 'upstream/master',
+    defaultsTo: null, // This option is required
```

Comment:
Default value will be null anyway

Reply:
Also, I would add that the option is required in the help text

Reply:
Yeah, I explicitly added a null so people wouldn't add a default later. I will update the help text, though

Reply:
done

Reply:
Makes sense!